onmt/translate/translator.py

```diff
@@ -1031,7 +1031,7 @@ def translate(
             phrase_table=phrase_table,
         )
 
-    def translate_batch(self, batch, src_vocabs, attn_debug):
+    def translate_batch(self, batch, batch_size, src_vocabs, attn_debug):
```

Comment:
This looks like a breaking change that might have unwanted consequences. Is it really necessary?